packages/in_app_purchase/in_app_purchase/example/lib/main.dart

```diff
@@ -438,6 +451,28 @@ class _MyAppState extends State<_MyApp> {
     });
   }
 
+  Future<void> confirmPriceChange() async {
+    if (Platform.isAndroid) {
+      final InAppPurchaseAndroidPlatformAddition androidAddition =
+          _inAppPurchase
+              .getPlatformAddition<InAppPurchaseAndroidPlatformAddition>();
+      var priceChangeConfirmationResult =
+          await androidAddition.launchPriceChangeConfirmationFlow(
+        sku: 'purchaseId',
+      );
+      if (priceChangeConfirmationResult.responseCode == BillingResponse.ok) {
+        // TODO acknowledge price change
+      } else {
+        // TODO show error
+      }
```

Comment:
@mvanbeusekom I'm not sure what to do here. Should I just add a print statement, leave these todo's or do you have an idea?

Reply:
Maybe we can simply show a snackbar message confirming the price change or showing the error?